lib/routes.dart

```diff
@@ -90,3 +97,26 @@ class RouteConfiguration {
     return null;
   }
 }
+
+class NoAnimationMaterialPageRoute<T> extends MaterialPageRoute<T> {
+  NoAnimationMaterialPageRoute({
+    @required WidgetBuilder builder,
+    RouteSettings settings,
+    bool maintainState = true,
```

Comment:
Are these every changed? They can be passed to super without passing to this constructor.

Reply:
Done!